### PR TITLE
Change the license key copy.

### PR DESCRIPTION
**Pull request by Witoso**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Change the license key details.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/getting-started/licensing/license-key-and-activation.md [hunk 1]
```diff
@@ -99,14 +99,14 @@ A token URL and other authentication methods will be assigned to this specific e
 
 ### Trial license key
 
-This key grants access to **all features**. It is valid for **14 days**. It does not consume editor loads, but editor is limited functionally (for example: session time, number of changes). It is **perfect for evaluating the platform** and all its features. It can be used only for evaluation purposes.
+This key grants access to **all features**. It is valid for **14 days**. It does not consume editor loads if you are on the usage-based billing plan, but editor is limited functionally (for example: session time, number of changes). It is **perfect for evaluating the platform** and all its features. It can be used only for evaluation purposes.
```
Comment (Witoso) [inline docs/getting-started/licensing/license-key-and-activation.md:102]: if you **are not** on the usage-based billing plan

Comment (gorzelinski) [inline docs/getting-started/licensing/license-key-and-activation.md:102]:
```suggestion
This key grants access to **all features.** It is valid for **14 days.** It does not consume editor loads if you are not on the usage-based billing plan. However, it limits the editor's functionality (like session time or number of changes). It is perfect for evaluating the platform and all its features. The key exists only for evaluation purposes.
```

Comment (gorzelinski) [reply docs/getting-started/licensing/license-key-and-activation.md:102]: It is a good practice to put punctuation marks in bold when the adjacent word is in bold.

File: docs/getting-started/licensing/license-key-and-activation.md [hunk 2]
```diff
@@ -99,14 +99,14 @@ A token URL and other authentication methods will be assigned to this specific e
...
 
 * **Features**: Grants access to all features and add-ons.
 * **Duration**: Valid for 14 days.
 * **Functionality**: The editor is limited functionally, such as session time and the number of changes allowed.
```
Comment (gorzelinski) [inline docs/getting-started/licensing/license-key-and-activation.md:106]:
```suggestion
* **Functionality**: The editor has limited functionality regarding session time and the number of changes allowed.
```

File: docs/getting-started/licensing/license-key-and-activation.md [hunk 3]
```diff
@@ -121,17 +121,19 @@ This key grants access to your subscription features. It does not consume editor
 	* The editor will show a banner informing it was launched for development purposes.
 * **Intended use**: Designed for development environments such as local work, continuous integration (CI), and end-to-end (E2E) tests.
 * **Usage limitation**: Must not be used for production environments.
-* **Editor loads**: Does not consume editor loads.
+* **Editor loads**: Does not consume editor loads regardless of the plan type.
```
Comment (gorzelinski) [inline docs/getting-started/licensing/license-key-and-activation.md:124]:
```suggestion
* **Editor loads**: It does not consume editor loads, regardless of the plan type.
```

File: docs/getting-started/licensing/license-key-and-activation.md [hunk 4]
```diff
@@ -99,14 +99,14 @@ A token URL and other authentication methods will be assigned to this specific e
...
-This key grants access to **all features**. It is valid for **14 days**. It does not consume editor loads, but editor is limited functionally (for example: session time, number of changes). It is **perfect for evaluating the platform** and all its features. It can be used only for evaluation purposes.
+This key grants access to **all features**. It is valid for **14 days**. It does not consume editor loads if you are on the usage-based billing plan, but editor is limited functionally (for example: session time, number of changes). It is **perfect for evaluating the platform** and all its features. It can be used only for evaluation purposes.
 
 * **Features**: Grants access to all features and add-ons.
 * **Duration**: Valid for 14 days.
 * **Functionality**: The editor is limited functionally, such as session time and the number of changes allowed.
 * **Intended use**: Ideal for evaluating the platform and all its features.
 * **Usage limitation**: Can only be used for evaluation purposes and not for production.
-* **Editor loads**: Does not consume editor loads.
+* **Editor loads**: Does not consume editor loads regardless of the plan type.
```
Comment (gorzelinski) [inline docs/getting-started/licensing/license-key-and-activation.md:109]:
```suggestion
* **Editor loads**: It does not consume editor loads, regardless of the plan type.
```

File: docs/getting-started/licensing/license-key-and-activation.md [hunk 5]
```diff
@@ -121,17 +121,19 @@ This key grants access to your subscription features. It does not consume editor
...
+This key grants access to your subscription features without imposing any operational limitations.
 
 * **Features**: Grants access to subscription features.
 * **Functionality**: The editor functions without any restrictions.
 * **Intended use**: Meant for production environments where the software is actively used by end-users.
 * **Usage limitation**: None specified.
-* **Editor loads**: Consumes editor loads.
+* **Editor loads**: Depending on your plan type:
+    * Consumes editor loads if you are on our usage-based billing plans.
+	* Does not consume editor loads if you are not on a plan with usage-based billing.
```
Comment (gorzelinski) [inline docs/getting-started/licensing/license-key-and-activation.md:136]:
```suggestion
    * It consumes editor loads if you are on our usage-based billing plans.
	* It does not consume editor loads if you are not on a plan with usage-based billing.
```

**Review by gorzelinski — COMMENTED** [review]
I added some suggestions

**Review by gorzelinski — APPROVED** [review]

**Pull request merged**